mimic/rest/auth_api.py

```diff
@@ -345,6 +345,120 @@ def validate_token(self, request, token_id):
+                    "id": "hybrid:123456",
+                    "name": "hybrid:123456",
+                }
+                response["access"]["user"] = {
+                    "id": "34",
+                    "name": "HybridThreeFour",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:creator",
+                               "description": "Monitoring Creator"},
+                              {"id": "2",
+                               "name": "creator",
+                               "description": "Creator"}],
+                    "RAX-AUTH:contactId": "34"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_limited_device_permission_holder"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:123456",
+                    "name": "hybrid:123456",
+                }
+                response["access"]["user"] = {
+                    "id": "56",
+                    "name": "HybridFiveSix",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:observer",
+                               "description": "Monitoring Observer"},
+                              {"id": "2",
+                               "name": "observer",
+                               "description": "Observer"}],
+                    "RAX-AUTH:contactId": "56"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_other_account_observer"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:654321",
+                    "name": "hybrid:654321",
+                }
+                response["access"]["user"] = {
+                    "id": "78",
+                    "name": "HybridSevenEight",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:observer",
+                               "description": "Observer"},
+                              {"id": "2",
+                               "name": "observer",
+                               "description": "Observer"}],
+                    "RAX-AUTH:contactId": "78"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_other_account_admin"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:654321",
+                    "name": "hybrid:654321",
+                }
+                response["access"]["user"] = {
+                    "id": "90",
+                    "name": "HybridNineZero",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:admin",
+                               "description": "Admin"},
+                              {"id": "2",
+                               "name": "admin",
+                               "description": "Admin"}],
+                    "RAX-AUTH:contactId": "90"
+                }
+
```

Comment:
@tpboudreau-rackspace since I know what this is getting used for inside MaaS, I understand the need for expediency. I don't want you to have to run a fork. If MaaS can promise to help us iterate off these features once Mimic supports the proper ingestion endpoints, I'm happy to land this change as-is.